cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java

```diff
@@ -39,17 +40,23 @@
      * @param port
      * @param ssl
      * @param apiKey
+     * @param isPersonal
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/642cc873e21caa7b431ae9f3a59ca3a661e27039